lib/net.js

```diff
@@ -130,7 +131,7 @@ function getNewAsyncId(handle) {
 }
 
 
-const debug = util.debuglog('net');
+const debug = debuglog('net');
```

Comment:
In combination with the above suggestion:

```suggestion
```

Reply:
@BridgeAR Ok